index.html

```diff
@@ -0,0 +1,293 @@
+            </figure>
+            <div class="project-info">
+              <div class="project-header">
+                <h2 class="project-title">Multi-Post Stories</h2>
+                <div class="header-details">
+                  <p class="client">CANOPY</p>
+                  <span class="separator-circle"></span>
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h2 class="title">About Myself</h2>
+          <p class="details">
+            I am a software developer! If you have a project you need coded, do not hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET US CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="image/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="image/linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+            <button class="btn-default" type="button">Get my bio-data</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="image/chev-dow.png" alt="" />
+                </button>
+              </div>
+              <div class="separator-line"></div>
+              <ul class="language-list">
+                <li>
+                  <img src="image/javascript.png" alt="JavaScript" />JavaScript</li>
```

Comment:
Optional

- Well done styling the about section as requested by the previous reviewer. However, there are images which broken so I'm approving the project but I expect you'll be able to handle such a miner thing 😉🙏🏻❤️